modules/plugins/languages/ts.nix

```
@@ -17,18 +17,35 @@
 
   cfg = config.vim.languages.ts;
 
-  defaultServers = ["ts_ls"];
+  defaultServers = [
+    "ts_ls"
+    "vue_ls"
+  ];
```

```suggestion
defaultServers = ["ts_ls" "vue_ls"];
```

Please avoid formatting with nixfmt or similar tools. While this line *is* subject to change, it generally produces too much noise and ugly diffs.